server/cmd_scan_test.go

```diff
@@ -174,3 +207,20 @@ func TestZSetScan(t *testing.T) {
 	}
 
 }
+
+func TestZSetScan(t *testing.T) {
+	c := getTestConn()
+	defer c.Close()
+
+	key := "scan_zset"
+	c.Do("ZADD", key, 1, "a", 2, "b")
+
+	if ay, err := goredis.Values(c.Do("XZSCAN", key, "0")); err != nil {
+		t.Fatal(err)
+	} else if len(ay) != 2 {
+		t.Fatal(len(ay))
+	} else {
+		checkScanValues(t, ay[1], "a", 1, "b", 2)
+	}
+
```

Comment:
ditto

Reply:
Sure thing, I think it was there in the original code, but I'll remove it either way.